kustomize/overlays/prod/kustomization.yaml

```diff
@@ -1,12 +1,18 @@
-namespace: event-bridge-prod
+namespace: jrota-test
 resources:
   - ../../base
   - all-in-one
+  - secrets
+  - shard
   - observability
 patchesStrategicMerge:
   - all-in-one/patches/deploy-config.yaml
   - all-in-one/patches/postgres-config.yaml
+  - shard/patches/deploy-config.yaml
 images:
   - name: event-bridge-all-in-one
     newName: quay.io/5733d9e2be6485d52ffa08870cabdee0/event-bridge-all-in-one
-    newTag: 711f0a6bcce14be5086977a1cb6cdc5d5d375ce4-jvm
+    newTag: 587c24c836a96fa44c7d0ea929c784721616e239-jvm
+  - name: event-bridge-shard-operator
+    newName: quay.io/5733d9e2be6485d52ffa08870cabdee0/fleet-shard
+    newTag: latest
```

Comment:
The idea is the following: 
1) we start with latest
2) When this PR gets merged new docker images is pushed to the quay.io repos
3) The job that creates a new PR for the kustomize project will update the images with the right tag

Reply:
@r00ta Flow looks good to me. I wonder if we should bring that job as a GH Action onto this repository now to make it visible. At the minute I think it's a bot somewhere in your personal account, so there is some "magic" happening somewhere :-)

Reply:
@robpblake created https://issues.redhat.com/browse/MGDOBR-149